### ISSUE TITLE
Update Directions CLI

### ISSUE THREAD

**Issue by perrygeo** [mapbox/mapbox-cli-py#109]
When https://github.com/mapbox/mapbox-sdk-py/pull/203 gets merged into `mapbox-sdk-py`, the CLI will need a corresponding update.

**Comment by wboykinm** [comment]
@perrygeo Any chance this explains why I'm currently getting `mapbox cli 'Directions' object has no attribute 'valid_instruction_formats'` when trying the `upload` endpoint?

**Comment by perrygeo** [comment]
@wboykinm What versions of `mapbox` and `mapboxcli` python modules do you have installed? There might be some incompatibilities between certain versions that we need to fix.

```
python -c 'import mapboxcli; print(mapboxcli.__version__)'
python -c 'import mapbox; print(mapbox.__version__)'
```

**Comment by wboykinm** [comment]
`0.7.0` and `0.16.0` respectively.

**Comment by perrygeo** [comment]
@wboykinm Unfortunately the 0.16.0 and 0.7.0 releases are not compatible. That's on us, sorry. 

We plan on getting a 0.8 release out next week. Until then, the temporary fix would be to downgrade to `mapbox-sdk-py==0.15.0`.

**Comment by perrygeo** [comment]
closed by #124
